modelci/hub/deployer/k8s_dispatcher/k8s_dispatcher.py

```diff
@@ -0,0 +1,143 @@
+    else:
+        raise RuntimeError(f'`{remote_storage}` currently is not supported.')
+
+    # for continer
+    deployment_conf = dict(config['deployment'])
+
+    deployment['name'] = deployment_conf['name']
+    deployment['namespace'] = deployment_conf['namespace']
+    deployment['replicas'] = deployment_conf['replicas']
+
+    engine: Engine = Engine[deployment_conf['engine'].upper()]
+
+    cuda, device_num = get_device(deployment_conf['device'])
+
+    env_objects = dict()
+    port_objects = dict()
+
+    env_objects['BATCH_SIZE'] = deployment_conf['batch_size']
+    if cuda:
+        env_objects['CUDA_DEVICE_ORDER'] = 'PCI_BUS_ID'
+        env_objects['CUDA_VISIBLE_DEVICES'] = device_num
+
+    if engine == Engine.TFS:
+        docker_tag = '2.1.0-gpu' if cuda else '2.1.0'
+        deployment['image'] = f'tensorflow/serving:{docker_tag}'
+        port_objects['tfs_http_port'] = 8501
+        port_objects['tfs_grpc_port'] = 8500
+        env_objects['MODEL_NAME'] = model_conf['local_model_name']
+    elif engine == Engine.TORCHSCRIPT:
+        docker_tag = 'latest-gpu' if cuda else 'latest'
+        deployment['image'] = f'mlmodelci/pytorch-serving:{docker_tag}'
+        port_objects['torchscript_http_port'] = 8000
+        port_objects['torchscript_grpc_port'] = 8001
+        env_objects['MODEL_NAME'] = model_conf['local_model_name']
+    elif engine == Engine.ONNX:
+        docker_tag = 'latest-gpu' if cuda else 'latest'
+        deployment['image'] = f'mlmodelci/onnx-serving:{docker_tag}'
+        port_objects['onnx_http_port'] = 8000
+        port_objects['onnx_grpc_port'] = 8001
+        env_objects['MODEL_NAME'] = model_conf['local_model_name']
+    elif engine == Engine.TRT:
+        if not cuda:
+            raise RuntimeError('TensorRT cannot be run without CUDA. Please specify a CUDA device.')
+
+        deployment['image'] = 'nvcr.io/nvidia/tensorrtserver:19.10-py3'
+        # ulimits currently can't be set on kubernetes
+        #Check open issue: https://github.com/kubernetes/kubernetes/issues/3595
+        deployment['trt_model_repo'] = model_conf['local_model_dir']
+        port_objects['trt_http_port'] = 8000
+        port_objects['trt_grpc_port'] = 8001
+        port_objects['trt_prometheus_port'] = 8002
+    else:
+        raise RuntimeError(f"Not able to serve model with engine `{deployment['engine']}`.")
+
+    env_object_list = generate_object_list(env_objects)
+    port_object_list = generate_object_list(port_objects)
+    if additional_environment:
+        env_object_list = env_object_list + generate_object_list(additional_environment)
+
+    deployment_content = template.render(
+        init_container=init_container,
+        init_env_object_list=init_env_object_list,
+        deployment=deployment,
+        env_object_list=env_object_list,
+        port_object_list=port_object_list
+    )
+
+    with open(output_file_path, 'w+') as output_target:
```

Comment:
Can you also update the readme.md under the deployer dir?

This PR contains no example or test cases.

Reply:
Added a readme